pages/en/lb3/Creating-a-database-schema-from-models.md

```diff
@@ -45,6 +45,10 @@ The following data sources support auto-migration:
 * [PostgreSQL](PostgreSQL-connector.html)
 * [SAP HANA](https://www.npmjs.org/package/loopback-connector-saphana) - Not officially supported; see [Community connectors](Community-connectors.html).
 * [SQL Server](SQL-Server-connector.html)
+* [IBM Cloudant](Cloudant-connector.html)
+* [IBM DB2](DB2-connector.html)
+* [IBM DashDB](DashDB.html)
+* [IBM DB2 for z/OS](DB2-for-z-OS.html)
```

Comment:
I know it's not related to the purpose of the PR.. I'm wondering if it organizes better to move the SAP HANA at the bottom, because that's the only community connector on the list.